### ISSUE TITLE
local history missing

### ISSUE THREAD

**Issue by 2019-05-10** [dbeaver/dbeaver#8688]
Most recently connected to #8617 I noticed again that there's no (local) history of editors/files.
 
I'd like to see the Eclipse feature "Compare with Local History" and access to the edit history in general to track changes and be able to revert to older versions.

Not sure if it is intentional or just nobody noticed it gone missing.

**Comment by uslss** [comment]
thanks for suggestion

**Comment by serge-rider** [comment]
Local history feature is there. 
It is available from context menu of project explorer or SQL editor:

![image](https://user-images.githubusercontent.com/6398845/81958186-476d1700-9616-11ea-8671-0defdbeeb9af.png)
![image](https://user-images.githubusercontent.com/6398845/81958229-5ce24100-9616-11ea-9a73-5375ce0f0aa1.png)

Maybe you mean some other Eclipse feature?

**Comment by 2019-05-10** [comment]
Confusing.
It's not there when you right click _in_ the editor:
![image](https://user-images.githubusercontent.com/50481865/82039714-b6458100-96a5-11ea-8d06-67248d931987.png)

**Comment by uslss** [comment]
@2019-05-10 what dbeaver version do you use?

**Comment by 2019-05-10** [comment]
> @2019-05-10 what dbeaver version do you use?

Currently
Version 7.1.3.202007192031

and here, the context menu now is even missing the 

Compare With 
Replace With

entries entirely!
See Screenshot:
![image](https://user-images.githubusercontent.com/50481865/88266158-fc831600-ccce-11ea-90f2-cb392c2bc4eb.png)

Given that I usually only use the "Database Navigator" view, the feature is not easily accessible.

**Comment by uslss** [comment]
@2019-05-10 feature is available, e.g. in Project Tab for the script
![image](https://user-images.githubusercontent.com/31996417/88268385-0a3e9880-ccdb-11ea-80c6-114d784c9f40.png)

**Comment by 2019-05-10** [comment]
Yes, we established that already, see serge-rider's comment on May 14.
And my comment on May 15.
The question is, why it isn't in the editor's context menu as it 

a) is in Eclipse
b) would be sensible

And as of now:
why has in the editor's context menu the entry "Compare With" disappeared entirely instead of just being extended?

**Comment by uslss** [comment]
catch you. thanks for clarification
